packages/interactivity-router/src/head.ts

```diff
@@ -41,12 +41,17 @@ export const updateHead = async ( newHead ) => {
  * Fetches and processes head assets (stylesheets and scripts) from a specified document.
  *
  * @async
- * @param {Document} doc          The document from which to fetch head assets. It should support standard DOM querying methods.
- * @param {Map}      headElements A map of head elements to modify tracking the URLs of already processed assets to avoid duplicates.
+ * @param {Document} doc               The document from which to fetch head assets. It should support standard DOM querying methods.
+ * @param {Map}      headElements      A map of head elements to modify tracking the URLs of already processed assets to avoid duplicates.
```

Comment:
In Interactivity API package we are removing the type from the jsdoc to only leave it in TS.